src/browser/components/atoms/Userimage/userimage.css

```diff
@@ -0,0 +1,4 @@
+.user-image {
```

Comment:
if you move the classname as I suggested above you could change this to `.user-image > img {`